src/druglab/pipe/blocks/filter.py

```
@@ -0,0 +1,314 @@
+        config.update({
+            "min_mw": self.min_mw, "max_mw": self.max_mw,
+            "min_logp": self.min_logp, "max_logp": self.max_logp
+        })
+        return config
+
+    def _process_item(self, item):
+        if item is None:
+            return False
+        from rdkit.Chem import Descriptors
+        
+        if self.min_mw is not None and Descriptors.MolWt(item) < self.min_mw: return False
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)